deltachat-ffi/deltachat.h

```diff
@@ -7231,6 +7237,21 @@ void dc_event_unref(dc_event_t* event);
 /// Used as the first info messages in newly created groups.
 #define DC_STR_NEW_GROUP_SEND_FIRST_MESSAGE 172
 
+/// "Member %1$s added."
+///
+/// Used as info messages.
+///
+/// `%1$s` will be replaced by the added member's name.
+#define DC_STR_MESSAGE_ADD_MEMBER 173
+
+/// "Your email provider %1$s requires end-to-end encryption which is not setup yet."
+///
+/// Used as info messages when a message cannot be sent because ot cannot be encrypted.
```

Comment:
```suggestion
/// Used as info messages when a message cannot be sent because of `cannot be encrypted`.
```

Reply:
thanks, gist was an "it", however :) i'll add a commit